### PR TITLE
Video Renderer

### PR DESCRIPTION
**Pull request by thylsky**
### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (what?)

### 🔗 Related issue link

Renderer for a Video block type

### 💡 Background and solution

Mostly reused Image component renderer. Styles were adjusted so the video is not cut.

### 📝 Changelog

Added Video block Renderer

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo/Example is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: index.jsx [hunk 1]
```diff
@@ -44,6 +45,7 @@ const Output = ({ data, style, config }) => {
       case 'paragraph':
         return <ParagraphOutput key={ index } data={ block.data } style={ style.paragraph || {}} config={ config.paragraph || {}} />;
       case 'image': return <ImageOutput key={ index } data={ block.data } style={ style.image || {}} config={ config.image || {}} />;
+      case 'video': return <VideoOutput key={ index } data={ block.data } style={ style.video || {}} config={ config.video || {}} />;
```
Comment (dev-juju) [inline index.jsx:48]: Please add VideoOutput to the export object at bottom of file

File: renderers/videoOutput/index.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  if (!data.withBorder) figureStyle.border = 'none';
+  if (!data.withBackground) figureStyle.backgroundColor = 'none';
+
+  const figcaptionStyle = config.disableDefaultStyle ? style.figcaption : { ...videoOutputStyle.figcaptionStyle, ...style.figcaption };
+
+  return (
+    <figure style={ figureStyle }>
+      <video style={ videoStyle } autoPlay={ data.autoPlay } muted={ data.muted } controls={ data.controls }>
+        <source src={ data.url } />
```
Comment (dev-juju) [inline renderers/videoOutput/index.jsx:40]: Please add info/warning message for browsers that do not support HTML5 video tag

**Comment by thylsky** [comment]
Comments resolved @BomdiZane 🙂

**Review by dev-juju — CHANGES_REQUESTED** [review]
Hello @thylsky ,

Please review comments and make appropriate adjustments so that we can merge your pull request.

We appreciate you contributions 🙏

**Review by dev-juju — APPROVED** [review]

**Pull request merged**